client/stylesheets/_toCat.scss

```diff
@@ -5326,6 +5333,29 @@ On click the images will be displayed at normal size to complete the effect
 
 /* End */
 
+.listingPayment {
+	display: block !important;
+	margin: -75px -100px 0 0;
+}
+
+.listingPayment li {
+	display: inline-block;
+	margin: 0;
+}
+
+.listingPayment li i {
```

Comment:
Merge rule `.listingPayment li i` with rule on line 5336<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `listingPayment` should be written in lowercase with hyphens